src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void exitMessage() {
+        System.out.println("Go away Anna");
+        System.out.println("O-kay bye......");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hi it's Anna!\nWhat do you need to do?");
+        Scanner in = new Scanner(System.in);
+
+        boolean exit = false;
```

Comment:
Preferably name boolean in such a way that implies it is boolean.

Reply:
Maybe use isExit to replace exit can imply a boolean